Vagrantfile

```diff
@@ -21,83 +21,276 @@ Vagrant.configure("2") do |config|
   config.vm.provider :virtualbox do |v|
     v.memory = 2048
     v.cpus = 2
+    v.customize ["modifyvm", :id, "--audio", "none"]
```

Comment:
unrelated to selinux?

Reply:
Unrelated to SELinux sure, but related to testing with VMs in VirtualBox on Mac/Linux in that they will setup an unnecessary audio device that can have detrimental effects on one's desktop/laptop computing experience.